src/index.js

```diff
@@ -39,8 +36,9 @@ type Namespace = {
 const namespace: Namespace = {
   async create(
     image: Image,
-    options: ?Options = defaultOptions,
+    options?: Options = defaultOptions,
   ): Promise<PaletteInstance> {
+    validation(image, options);
```

Comment:
I think this should be a verb - `validate` to express the intent to execute this subroutine.